<ui version="4.0" >
 <class>crashWindow</class>
 <widget class="QWidget" name="crashWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Tribe has crashed due to a fatal error. We are extremely sorry about that, as this should not happen. However, Tribe collected some important informations about the crash that will be extremely useful to developers for fixing this problem. The fastest way to report this, if you have an active internet connection, is clicking "Send report to developers". Otherwise, you can copy/paste the text below and send it to bugs@chakra-project.org. No personal informations are included, you can see the log for details.</string>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KTextBrowser" name="textBrowser" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="copyButton" >
       <property name="text" >
        <string>Copy to clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="sendButton" >
       <property name="text" >
        <string>Send report to developers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
